scipy/ndimage/_interpolation.py

```diff
@@ -654,6 +654,117 @@ def shift(input, shift, output=None, order=3, mode='constant', cval=0.0,
     .. versionadded:: 1.6.0
         Complex-valued support added.
 
+    Examples
+    --------
+    A standard call to `shift` will shift ``input`` values by the specified
+    ``shift`` number of indices in along all dimensions. Fractional values
+    of ``shift`` are supported, and the output values are interpolated.
+    
+    Values in the first ``shift`` indices in each dimension will be filled 
+    by ``0`` using  the default ``constant`` ``mode`` and ``cval`` of
+    ``0.0``.
```

Comment:
Again would prefer if this used language consistent with the description of `mode`